### DISCUSSION TITLE
Accedere a step successivo solo se si accettano i termini di servizio

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#3888] [Q&A] [answered]
Buongiorno, 
avrei necessità di far accedere i segnalanti allo step _2 - Questionario_ solo se nello _step 1 - Informativa privacy_, questi accettano i termini di servizio (spuntano il flag). 
C'è un modo per farlo oppure non è previsto?

**Comment by evilaliv3** [comment, marked as answer]
Certamente si, è previsto.

Basta che impostiate come obbligatorio rispondere alla domanda e il segnalante che prema "Successivo" senza aver spuntato la casella non potra andare allo step successivo. Questo è valido per ogni domanda obbligatoria.

Il segnalante potra tramite i tab di navigazione procedere comunque a vedere gli step successivi ma non potra completare l'invio della segnalazione salvo aver risposto a tutte le domande obbligatorie.

**Reply by maelaborazioni** [reply]
Perfetto, grazie mille!
